service/api_patch.go

```diff
@@ -53,12 +53,13 @@ func (as *APIServer) submitPatch(w http.ResponseWriter, r *http.Request) {
 	if patchString == "" {
 		patchString = data.PatchString
 	}
+	patchString = strings.TrimSpace(patchString)
```

Comment:
are we concerned for patches that contain only space?

Reply:
I'm not particularly concerned about that: don't diffs usually contain lines before and after also? I was mostly trying to circumvent something weird happening where the patch string wasn't empty but should be. If you think that check is overkill I don't mind dropping it.

Reply:
Failing tests have convinced me to drop it :)